bundle/2.0.0/metadata/annotations.yaml

```diff
@@ -4,9 +4,9 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: skupper-operator
-  operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: alpha,stable,stable-1,stable-1.8
+  operators.operatorframework.io.bundle.channel.default.v1: stable-2
+  operators.operatorframework.io.bundle.channels.v1: stable-2,stable-2.0
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.4.0+git
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
-  com.redhat.openshift.versions: v4.6
+  com.redhat.openshift.versions: v4.17
```

Comment:
Maybe I am mistaken, but I believe using 4.6 here tells that it is compatible with 4.6+,
not sure if we should use 4.17. Do you know anything different about it?
https://docs.openshift.com/container-platform/4.17/operators/operator_sdk/osdk-working-bundle-images.html#osdk-control-compat_osdk-working-bundle-images